src/sentry/api/endpoints/event_ai_suggested_fix.py

```diff
@@ -21,6 +22,8 @@
 
 openai.api_key = settings.OPENAI_API_KEY
 
+openai_policy_check = Signal()
```

Comment:
i wonder if it would be better to impl this as a service, and define a new service backend in getsentry

Reply:
Yeah I was thinking about it but I was pretty lazy.